install/nginx

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -e
+
+if [[ $# != 2 ]]
+then
+		echo "Usage: "
+		echo "	$0 userName fullURLToSite"
+		exit 1
+fi
+
+userName=$1
+FQDN=$2
+configFile=/etc/nginx/sites-available/$FQDN
+
+scriptDir="$(dirname $0)"
+cd $scriptDir
+# Read config file with paths to WP-installs and usernames
+source /usr/local/default/etc/wp-mass-admin/wpma.conf
+if [[ -e /usr/local/etc/wp-mass-admin/wpma.conf ]]; then
+    source /usr/local/etc/wp-mass-admin/wpma.conf
+fi
+
+if [[ "$useHttps" == "yes" ]]
+then
+	./use-https $userName $FQDN
+else
+	./no-https $userName $FQDN
+fi
+
```

Comment:
To me it looks like the user is only ever able to choose whether or not to use HTTPS for an install in the global config file and not on an invocation basis. I would say that in our situation in CYD we want the default to be HTTPS but with the option of opting out for the setup of some specific sites.